pyuvsim/analyticbeam.py

```diff
@@ -42,23 +42,19 @@ class AnalyticBeam(object):
     Directly calculates jones matrices at given azimuths and zenith angles
     from analytic functions.
 
-    Supports uniform (unit response in all directions), gaussian, and Airy
-    function beam types.
-
     Supported types:
 
         * Uniform beam : Unit response from all directions.
-        * Airy: An Airy disk pattern (the 2D Fourier transform of a circular aperture of width
-        given by `diameter`)
+        * Airy: An Airy disk pattern (the 2D Fourier transform of a circular aperture of
+          width given by `diameter`)
         * Gaussian : A peak-normalized gaussian function.
```

Comment:
I think you need to take out the space before the colon in this line. The space is causing RTD to render it differently than the rest.

Reply:
Yup, did that.